packages/reassure-cli/src/commands/measure.ts

```diff
@@ -56,62 +60,72 @@ export async function run(options: MeasureOptions) {
     { shell: true, stdio: 'inherit', env: { ...process.env, OUTPUT_FILE: outputFile } }
   );
 
-  console.log('');
+  log('');
 
   if (spawnInfo.status !== 0) {
-    console.error(`❌  Test runner (${testRunnerPath}) exited with error code ${spawnInfo.status}`);
+    error(`❌  Test runner (${testRunnerPath}) exited with error code ${spawnInfo.status}`);
     process.exitCode = 1;
     return;
   }
 
   if (existsSync(outputFile)) {
-    console.log(`✅  Written ${measurementType} performance measurements to ${outputFile}`);
-    console.log(`🔗 ${resolve(outputFile)}\n`);
+    log(`✅  Written ${measurementType} performance measurements to ${outputFile}`);
+    log(`🔗 ${resolve(outputFile)}\n`);
   } else {
-    console.error(`❌  Something went wrong, ${measurementType} performance file (${outputFile}) does not exist\n`);
+    error(`❌  Something went wrong, ${measurementType} performance file (${outputFile}) does not exist\n`);
     return;
   }
 
   if (options.baseline) {
-    console.log("Hint: You can now run 'reassure' to measure & compare performance against modified code.\n");
+    warn("Hint: You can now run 'reassure' to measure & compare performance against modified code.\n");
```

Comment:
Why warn?

Reply:
Visibility and styling. `log` will be shown only in `--verbose` and `warn` shows always unless you run with `--silent`. I thought that;s a more important log than most general procedural logs.